src/components/quiz-question/quiz-question-wrapper.tsx

```diff
@@ -72,8 +72,12 @@ export const QuizQuestionWrapper = withText(translates)((props: QuizQuestionWrap
   useEffect(() => {
     setSelected(getSelected(qui));
     setIsLoading(false);
+    if(qui.q.questionType === KalturaQuizQuestionTypes.Reflection){
```

Comment:
can we use condition:
```
      if (qui.a || qui.q.questionType === KalturaQuizQuestionTypes.Reflection) {
        continueButtonRef.current?.focus();
      }
```
to avoid duplication of `continueButtonRef.current?.focus();`